assets/javascripts/modules/analytics/omniture.js

```diff
@@ -5,10 +5,25 @@ define([
 ], function ($, bean) {
     'use strict';
 
-    var dataTrackingClickableElements = ['a', 'button', 'input'],
-        omniture, s;
+    var omniture, s;
 
-    function sendEvent(prop17, pageName, products) {
+
+    function bindLinkTracking() {
+        $('a[data-link-tracking]').each(function (domElem) {
```

Comment:
Its OK for now because now a elements are the only one firing these events. If in the future other elements are used to fire these events then I think we should support at that time, but not before.